Makefile

```diff
@@ -17,25 +22,34 @@ STACKER_BUILD_UBUNTU_IMAGE?=$(STACKER_DOCKER_BASE)ubuntu:latest
 LXC_CLONE_URL?=https://github.com/lxc/lxc
 LXC_BRANCH?=stable-5.0
 
-stacker: stacker-dynamic
-	./stacker-dynamic --debug $(STACKER_OPTS) build \
+LEVEL1_STACKER ?= ./stacker-dynamic
```

Comment:
maybe call this BOOTSTRAP_STACKER

Reply:
STAGE1_STACKER?

Reply:
huh. i +1'd to the STAGE1_STACKER and then it changed to STACKER_BOOTSTRAP.  i like the STAGE1_STACKER, just pushed that.